DuckDuckGo/Fire/ViewModel/FirePopoverViewModel.swift

```diff
@@ -52,7 +52,8 @@ final class FirePopoverViewModel {
          fireproofDomains: FireproofDomains,
          faviconManagement: FaviconManagement,
          initialClearingOption: ClearingOption = .allData,
-         tld: TLD) {
+         tld: TLD,
+         contextualOnboardingStateMaching: ContextualOnboardingStateUpdater) {
```

Comment:
```suggestion
         contextualOnboardingStateMatching: ContextualOnboardingStateUpdater) {
```

Reply:
Why matching?

Reply:
I thought you meant matching pardon. But it probably needs to be updated to `contextualOnboardingStateMachine` instead of `contextualOnboardingStateMaching` ?